### PR TITLE
Add limit attachments validation. Can be configured with environment …

### PR DESCRIPTION
**Pull request by cekk**
…variable.

This prevent big files submissions and problems with gmail (can't send attachments > 25mb)

### CODE REVIEW

**Comment by erral** [comment]
I add @UnaiEtxaburu as a reviewer, he has been doing this kind of checks in our project.

File: src/collective/volto/formsupport/restapi/services/submit_form/post.py [hunk 1]
```diff
@@ -245,7 +295,7 @@ def filter_parameters(self):
 
     def send_mail(self, msg, encoding):
         host = api.portal.get_tool(name="MailHost")
-        host.send(msg, charset=encoding)
+        host.send(msg, charset=encoding, immediate=True)
```
Comment (mamico) [inline src/collective/volto/formsupport/restapi/services/submit_form/post.py:298]: @cekk Why this change was necessary?

Comment (cekk) [reply src/collective/volto/formsupport/restapi/services/submit_form/post.py:298]: because without this, i can't catch mail send exceptions like the one from gmail.
I haven't checked it, but it seems that without immediate=True, exceptions are handled by the mailhost and not raised to me (but logged)

Comment (mamico) [reply src/collective/volto/formsupport/restapi/services/submit_form/post.py:298]: Okay. But be careful. AFAIK Mailhost is transactional, with your change it is possible that the email will be sent, but for other problems (ConflictError or any other exception) the record will not be saved in the db. This is probably not a problem here. Consider making a comment to explain your choice.

Comment (cekk) [reply src/collective/volto/formsupport/restapi/services/submit_form/post.py:300]: ok, comment added

**Comment by cekk** [comment]
@UnaiEtxaburu what do you think about this pr?

**Review by erral — APPROVED** [review]
Sorry, I thought it was a frontend thing, that's why I asked Unai to check this, but it's all backend stuff :wolf: 

I have been checking this and it works as expected
